org.wso2.codequalitymatrices/pom.xml

```diff
@@ -20,20 +20,21 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-
-    <groupId>kasun.codeQualityMatrices</groupId>
+    <parent>
+        <groupId>org.wso2</groupId>
+        <artifactId>wso2</artifactId>
+        <version>5</version>
+    </parent>
+    <groupId>org.wso2.codeQualityMatrices</groupId>
     <artifactId>CodeQualityMatrices</artifactId>
```

Comment:
I think the practice is to use lower case, not uppercase and shall we use hyphen(-) to separate out different words, for ex: code-quality-matrices?